information-system/provenance/overview/what-are-entities.md

```diff
@@ -8,14 +8,58 @@ parent: Provenance Overview
+
+Registering provenance records with references to the registered entity provides the IS with the ability to create provenance graphs. Users of the IS are then able to explore and query these provenance graphs. 
+
+## 2. What entity types are used in the RRAP M&DS IS?
+
+### 2.1. Underlying conceptual model
+
+The underlying conceptual model for entities in the RRAP M&DS IS uses PROV-O. As depicted in the image below, 3 core concepts are defined: 
+* Entity
+* Agent
+* Activity
+
+![PROV-O Conceptual Model](https://upload.wikimedia.org/wikipedia/commons/thumb/7/7f/W3C_PROV_Data_Model.svg/661px-W3C_PROV_Data_Model.svg.png "PROV-O Conceptual Model")
+
+"An entity is a physical, digital, conceptual, or other kind of thing with some fixed aspects; entities may be real or imaginary" [1]. An *Entity* represents a thing in the world. An entity may be derived from other entities, and may be generated by an *Activity* that used a set of entities. 
+
+An *Activity* "is something that occurs over a period of time and acts upon or with entities; it may include consuming, processing, transforming, modifying, relocating, using, or generating entities". [1]
+
+"An *Agent* is something that bears some form of responsibility for an activity taking place, for the existence of an entity, or for another agent's activity [1]. This may be a person or a piece of software that is executed and associated with the activity that generates the Entity. It provides a way to attribute the activity and generation of the Entity to that agent. 
+
+
+### 2.2. RRAP M&DS IS Conceptual Model
+
+In RRAP M&DS, the PROV-O model is extended to enable capture of provenance information relating to model run workflows and other related information (input data, associated modellers, outputs data, modelling software/processes) (see figure below).
+
+| Provenance model used in M&DS using PROV-O |
+|:-:|
+|<img src="../../../assets/images/provenance/provenance-abstract-model.jpg" alt="drawing" width="600"/>|
+
+The specialised types used in M&DS IS are provided below:
+
+| Type        | Sub-type           | Description  |
+| ------------- |-------------| -----|
+| Entity      | Dataset Template | A defined template of a dataset and its anticipated set of resources. |
+| Entity      | Dataset | Structured collection of data/files associated with a body of work. |
+| Entity      | Model Run Workflow Template |  A defined template of a model run workflow and its associated set of input and output templates, and annotations. |
+| Agent      | Organisation | "An organization is a social or legal institution such as a company, society, etc." [1] |
+| Agent      | Person | "Person agents are people" [1] |
```

Comment:
What about software agents?

Reply:
Hmm, we haven't defined a subtype called "software agents" yet. I think we'd have one down the track as users start to have software agents, but I haven't seen a pull for it yet. @PeterBaker0 - have you had any conversations about this yet with users?

Reply:
No - I think the main pull has been for the ability to more specifically attribute responsibility/credit e.g. 'collaborators'.